reports/final/3-Technical/2-Details/15-Demo.md

```diff
@@ -0,0 +1,55 @@
+### Demo {#sec-3-2-15-1}
+
+#### Demo Layout {-}
+
+[demo-layout]: ./images/DemoLayout.png "Demo Layout" {#fig-demo-layout}
```

Comment:
This line is still wrong. It needs to be moved to the bottom of the List of Figures file with the other links